packages/trpc/server/routers/viewer/slots.ts

```diff
@@ -372,6 +437,9 @@ export async function getSchedule(input: z.infer<typeof getScheduleSchema>, ctx:
 
   const timeSlots: ReturnType<typeof getTimeSlots> = [];
 
+  const organizerTimeZone =
+    eventType.timeZone || eventType?.schedule?.timeZone || userAvailability?.[0]?.timeZone;
```

Comment:
I'm not sure userAvailability?.[0] works; I think you need to involve the defaultSchedule here